docs/user-guide/service-accounts.md

```diff
@@ -25,15 +25,42 @@ When they do, they are authenticated as a particular Service Account (e.g.
 
 ## Using the Default Service Account to access the API server.
 
-When you create a pod, you do not need to specify a service account.  It is
-automatically assigned the `default` service account of the same namespace.  If
-you get the raw json or yaml for a pod you have created (e.g. `kubectl get
-pods/podname -o yaml`), you can see the `spec.serviceAccount` field has been
+When you create a pod, if you do not specify a service account, it is
+automatically assigned the `default` service account in the same namespace.
+If you get the raw json or yaml for a pod you have created (e.g. `kubectl get pods/podname -o yaml`),
+you can see the `spec.serviceAccountName` field has been
 [automatically set](/docs/user-guide/working-with-resources/#resources-are-automatically-modified).
 
-With service accounts, you can access the API inside the pod using a proxy or with a client library,
+You can access the API from inside a pod using automatically mounted service account credentials,
```

Comment:
Maybe note that in cluster that enable authorization, service accounts have no permissions when accessing the API and must explicitly be granted them? It's mentioned a little later, but might help keep someone from freaking out here.